portality/forms/validate.py

```diff
@@ -110,56 +113,14 @@ def __make_optional(self, form, field):
         raise validators.StopValidation()
 
 
-class ExtraFieldRequiredIf(OptionalIf):
```

Comment:
I'm assuming this is unused

Reply:
Yup, the only place it appears is here where it is declared